images/conduit/opensearch/Dockerfile

```diff
@@ -10,4 +10,10 @@ COPY entrypoint.sh /entrypoint.sh
 ADD https://artifacts.opensearch.org/opensearch-clients/opensearch-cli/opensearch-cli-${OPENSEARCH_CLI_VERSION}-linux-arm64.zip /opensearch-cli.zip
 RUN cd /root && unzip /opensearch-cli.zip
 
+# AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip" && \
```

Comment:
You can save yourself from also installing `curl` by using the `ADD` directive:

```Dockerfile
ADD https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip ./awscliv2.zip
```

Reply:
Good to know. It's also right there on the line above, don't know how I didn't see it!

Reply:
The other advantage with this is that if the cache invalidation headers for the URL haven't changed, docker will not re-run the add command.

Reply:
I've swapped to using the ADD command but curl is required later so will still need to install.